### ISSUE TITLE
Can phylodivnet be added to betaDiversity vignette?

### ISSUE THREAD

**Issue by msmcfarlin** [adw96/DivNet#107]
Hello!

Thank you for the amazing packages breakaway and DivNet.

I've been exploring the vignettes and functions in these packages and came across the function `phylodivnet`. I didn't see it in any of the vignettes but it appears to be for unifrac distance calculation for evaluating beta diversity? Can an example with `phylodivnet` be added to the beta diversity vignette?

I tried running `phylodivnet` with a phyloseq object that includes a phylogenetic tree and a diagonal design matrix like I would when running `divnet` for beta diversity estimation and received the error... 
_Error in dimnames(x) <- dn : length of 'dimnames' [2] not equal to array extent_

I'm running this on a dataset that I collapsed to the phylum level. Could this error be related to that? That the number of taxa doesn't reflect the tips of the phylogenetic tree?

Thanks!

**Comment by adw96** [comment]
Hi @msmcfarlin -- unfortunately we looked into this a few years ago and doing this properly was computationally infeasible. At this time it is triaged; sorry. I won't close your issue, but just so you know, we won't be prioritizing this at the moment. 

Thanks for understanding -- there's so many cool stats things that we're working on instead!

**Comment by mooreryan** [comment]
Out of curiosity, did you find the unifrac distance calculation itself was the bottleneck? (Say as compared to other beta diversity measures like Euclidean or bray Curtis?)

**Comment by adw96** [comment]
Actually, we decided that if we wanted to truly estimate the uncertainty in e.g., UniFrac, we should account for the uncertainty in estimating the tree as well... and that was the bottleneck. I don't like underestimating variance, so we decided to leave this alone and focus our phylogenetic uncertainty investigations elsewhere -- @svteichman is making amazing progress on this! (but it won't look like DivNet) Happy to answer more questions @mooreryan , and @paulinetrinh can chime in too if there's anything I've forgotten.
